### DISCUSSION TITLE
Short Feedback

### DISCUSSION THREAD

**Discussion by MoritzIsVisiting** [snipe/snipe-it#15546] [General]
Hi, we have tried your solution for our company. I really like the solution, the prices for hosting are fair and affordable. 

But unfortunately we have to cancel our subscription again because we miss a "simple" feature: custom fields on licenses.

Background: We are ISO 27001 certified and have to keep an asset inventory. Since we are an e-commerce company with almost no assets, our asset inventory is quite simple. Most of it is software licenses. We have to make certain classifications for the assets, which I would like the respective owner to enter in the tool. I also think it's great that there are many different options for configuring the custom fields for normal assets.

The price of an asset management solution from another company is just too high for our simple use case, so I found your solution pretty perfect (hosting in Europe, automatic backups, SAML and Jamf integration, SCIM). But without the ability to create custom fields that can be exported, it's not feasible for us. 

Thank you for making such a good solution open source and providing easy and affordable hosting. Maybe we will meet again :-)

**Comment by snipe** [comment]
Thanks for that feedback. We're pretty aware that folks want custom fields on other things, (see https://github.com/snipe/snipe-it/issues/4261) and they're in the works, it's just really hard to implement without breaking other parts of the system. If you'd like, you can subscribe to that issue and get notified when updates are made.
